nntrainer/graph/network_graph.cpp

```diff
@@ -704,6 +704,7 @@ NetworkGraph::updateRunContext(std::shared_ptr<Manager> &manager,
    */
   // const RunLayerContext &run_context = lnode->getRunContext();
   lnode->updateRunContext(RunLayerContext(
+    // TODO: update weights spec for trainable based on layer trainable prop
```

Comment:
Shouldn't this must be done inside getWeightsSpec already before come to this spot.

Reply:
It depends on when we allow to change trainable property. Maybe this should be done in the end of after finalize() in layer_node.h

Reply:
I will add its support in #1312 or its next PR.

Reply:
Okay